docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/README.md

```diff
@@ -0,0 +1,140 @@
+# {{ cookiecutter.app_verbose_name }}
+
+## Cookiecutter Notes
+
+### Preparing for Docker
+
+Before building the Docker container, initialize the database:
+
+```bash
+docker-compose run --rm app python manage.py makemigrations
+```
+
+```bash
+docker-compose run --rm app python manage.py migrate
+```
+### Initial Docker Run
+
+When running `docker-compose up` for the first time, you may run into a database connection
+error that causes Docker to hang. Simply force quit Docker with `Ctrl-C` and re-run
+`docker-compose up`.
+
+### Initial Wagtail CMS Content
+
+The `fixtures/` directory has been intentionally left empty. Including `initial_cms_content.json`
+and `initial_cms_content_custom_pages.json` would require creating a Django admin account
+in advance which would introduce a security vulnerability if the admin password was not
+updated.
+
+After initializing the database, follow the instructions below to generate the initial CMS content.
```

Comment:
It's a little confusing to have this so high up. What do you think about merging it, along with the section on adding admin users, into the "Initial CMS content" section at the bottom? Something like: "This template does not include a default user or starter pages in the fixtures directory, so you'll need to create an admin user and initialize initial content as needed by your site... To save your default content..."